lib/models/config.coffee

```diff
@@ -100,7 +100,6 @@ getConfigModel = (deps, opts) ->
 			method: 'GET'
 			url: '/device-types/v1'
```

Comment:
I assume there are no breaking changes in the API for private types, right?

Reply:
No there are no breaking changes.
We were already able to send the auth token but that didn't have any effect so far, so it wasn't sent, so that we are explicit that it isn't required.
Now the auth token is optional, when present it's used to resolve the user permissions on private device types.
Let me add a comment about this.